cobrawap/pipeline/stage04_wave_detection/scripts/plot_waves.py

```diff
@@ -11,11 +11,13 @@
 from utils.neo_utils import analogsignal_to_imagesequence
 
 CLI = argparse.ArgumentParser()
-CLI.add_argument("--data", nargs='?', type=str)
-CLI.add_argument("--output", nargs='?', type=str)
+CLI.add_argument("--data", nargs='?', type=str,
+                 help="path to input data in neo format")
+CLI.add_argument("--output_dir", nargs='?', type=str,
+                 help="path to output directory")
 CLI.add_argument("--img_name", nargs='?', type=str)
```

Comment:
very very minor, just to be consistent with other changes introduced above, assume that each argument line has a "help" option

Reply:
(I inserted a suggestion, with the "empty" help option in a new line, to be coherent with what done above, but for short lines can also be put in the same line)

Reply:
Yes, right. I could add help="" where still missing, so to have it for every single command-line arg.